test/mail_with_ldap.bats

```diff
@@ -51,6 +51,8 @@ function setup_file() {
     -e DOVECOT_TLS=no \
     -e DOVECOT_PASS_FILTER="(&(objectClass=PostfixBookMailAccount)(uniqueIdentifier=%n))" \
     -e DOVECOT_USER_FILTER="(&(objectClass=PostfixBookMailAccount)(uniqueIdentifier=%n))" \
+    -e DOVECOT_USER_ATTRS="=uid=%{ldap:UID},=gid=5000,=home=/var/mail/%Ln,=mail=maildir:~/Maildir" \
```

Comment:
I think that by adding Maildir here, the path to the mail folder changed to `/var/mail/<domain>/<user>/Maildir` which led to the test suite error. (I'm using this scheme too but the test suite might not have been doing it)

Reply:
Thank you for contributing in this, @Murazaki! Something still missing apparently, could you check it? I have been working on another important issue in the last few days, maybe we can close this permission issue also if we work on this together.

Reply:
Hi Mohammed. I tried here but couldn't finish either. There might be a file permission issue: 
https://github.com/Murazaki/docker-mailserver/tree/patch-1